contracts/base/AssetController.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity 0.5.17;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20Detailed.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/utils/Address.sol";
+import "../interfaces/ISwapper.sol";
+import "../util/AddressArrayLib.sol";
+import "../interfaces/IAssetController.sol";
+
+contract AssetController is IAssetController {
+    using Address for address;
+    using AddressArrayLib for AddressArrayLib.AddressArray;
+
+    ISwapper public uniswapController;
+
+    ICompoundComptroller public compoundComptroller;
+
+    AddressArrayLib.AddressArray private assets;
+
+    mapping(string => address) public assetBySymbol;
+
+    function getAssets() external view returns (address[] memory) {
+        return assets.array;
+    }
+
+    function addAsset(address asset) external {
+        assets.add(asset);
+        assetBySymbol[ERC20Detailed(asset).symbol()] = asset;
+    }
```

Comment:
Lets also update `Settings` to remove the `private` assets array and use this for adding/getting assets.